### ISSUE TITLE
Binary missing (replaced by Git LFS text file)

### ISSUE THREAD

**Issue by jacobgkau** [pop-os/amd-ppt-bin#4]
I'm currently seeing a text file with the following contents in `/usr/bin/amd-ppt`:

```
jacob@pop-os:~$ cat /usr/bin/amd-ppt
version https://git-lfs.github.com/spec/v1
oid sha256:8ff62369fba576c79ab810a990b9b9ae992ca2a3c949b34fe5df58cd822ed130
size 260384
```

I am seeing overdraw occurring on the serw12 on battery power, presumably because the actual binary is missing.

**Comment by jackpot51** [comment]
Any idea when this happened?

**Comment by jacobgkau** [comment]
> Any idea when this happened?

I'm not sure. The overdraws were not happening when I was testing the keyboard hang in mid-November (I was using similar stress commands at that time.) If I had to guess, it might have been when build server keys were consolidated in late December.

I am seeing this on numerous Pop!_OS installations right now, so it's not a local installation problem or anything.

**Comment by jackpot51** [comment]
Probably fixed in https://github.com/pop-os/amd-ppt-bin/commit/1b805935ce4cf671a06e12252d606632dff130bf

**Comment by jackpot51** [comment]
@jacobgkau can you check if master staging has this fixed?

**Comment by jacobgkau** [comment]
> @jacobgkau can you check if master staging has this fixed?

Yes, this is fixed in master staging with `1b80593`.
